### DISCUSSION TITLE
passwd

### DISCUSSION THREAD

**Discussion by Bauer0102** [ALKindi-Wilagabrata/DELL-3421#8] [General]
What´s password to extract EFI?
Thanks

**Comment by alkindivv** [comment]
On Thu, 10 Nov 2022 at 06.01 Bauer0102 ***@***.***> wrote:

> What´s password to extract EFI?
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alkindivv/DELL-3421-BigSur/discussions/8>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APYUYH53IW2L4RPFJP3RIQ3WHQUL5ANCNFSM6AAAAAAR36G3D4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
Sorry its my client EFI and he pay me to made it, and its not for free for
everyone sorry

**Reply by Bauer0102** [reply]
ok, but it´s possible, do you help or i need to pay?

**Reply by alkindivv** [reply]
you'll need to pay
